app/templates/components/hyper-table.hbs

```diff
@@ -78,7 +78,8 @@
           {{/if}}
 
           <div class="hypertable__column">
-            {{hyper-table/cell column=column header=true manager=manager}}
+            {{hyper-table/cell
+              column=column header=true manager=manager sticky=true}}
```

Comment:
Just to understand, the `sticky` property is only apply to first column and can not be apply to other ones ?

Reply:
yep ! but i realise I could pass something lik `allowsFiltering` on the columns too. that way we could use it on all columns potentially